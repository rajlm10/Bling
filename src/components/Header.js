import React from 'react'
import {BrowserRouter,Route,Switch,Link,NavLink} from 'react-router-dom'


const Header=()=>{
    return(
        <header>
            <h1>Bling</h1>
            <NavLink to="/" activeClassName="is-active" exact={true}>Dashboard</NavLink>
            <NavLink to="/create" activeClassName="is-active">Add Expense</NavLink>
            <NavLink to="/edit" activeClassName="is-active">Edit Expense</NavLink>
            <NavLink to="/help" activeClassName="is-active">Help</NavLink>
        </header>
    )
}

export default Header