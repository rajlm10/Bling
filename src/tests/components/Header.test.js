import React from 'react'
import {shallow} from 'enzyme'
import Header from '../../components/Header'

 test('render header',()=>{
    const wrapper=shallow(<Header/>)
    expect(wrapper).toMatchSnapshot()


 })