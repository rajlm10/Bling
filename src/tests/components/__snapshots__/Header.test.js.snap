// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render header 1`] = `
<header>
  <h1>
    Bling
  </h1>
  <[object Object]
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/create"
  >
    Add Expense
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/edit"
  >
    Edit Expense
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/help"
  >
    Help
  </[object Object]>
</header>
`;
